cmd/piped/README.md

```diff
@@ -15,16 +15,22 @@
 - `make run/piped`: runs Piped locally (for more information, see [here](#how-to-run-piped-agent-locally)).
 - `make gen/code`: generate Go and Typescript code from protos and mock configs. You need to run it if you modified any proto or mock definition files.
 
-For the full list of available commands, please see the Makefile at the root of repository.
+For the full list of available commands, please see the Makefile at the root of the repository.
+
+## Setup Control Plane
+
+1. Prepare Control Plane that piped connects. If you want to run a control plane locally, see [How to run Control Plane locally](https://github.com/pipe-cd/pipecd/tree/master/cmd/pipecd#how-to-run-control-plane-locally).
+
+2. Access to Control Plane console, go to Piped list page and add a new piped. Then, copy generated Piped ID and key for `piped-config.yaml`
 
 ## How to run Piped agent locally
 
-1. Prepare the piped configuration file `piped-config.yaml`
+1. Prepare the piped configuration file `piped-config.yaml`. You can find an example config [here](https://github.com/pipe-cd/pipecd/tree/master/cmd/piped/piped-config.yaml).
```

Comment:
@kentakozuka Sorry, I mean, lets add the content of the example `piped-config.yaml` file in this file instead of making it a separate file 🙏